### DISCUSSION TITLE
Add `bun` and `bunx` run commands in the expo documentation

### DISCUSSION THREAD

**Discussion by tusharhqq** [expo/expo#30258] [Proposals and Ideas]
i have been using bun and bunx for a few months now, and everything when i copy any run command from expo website i need to edit that command from `npm` to `bun` and `npx`  to `bunx` 
![image](https://github.com/expo/expo/assets/79051850/6ae5d51d-76f4-41ff-8605-a48710eca9d8)

But there is another argument can also be given that why not add `bun`, if `bun` is being added then all the other one's should also be added like `pnpm`, `deno` etc ... and that will make expo website cluttered and unpleasing to the eyes and will make it worse. I really like the current look and feel of the website

**Comment by softwarebyze** [comment]
I just noticed expo agent using `bunx expo add ...`

![image](https://github.com/user-attachments/assets/c26b41f3-3565-4ff3-8b92-e6797c3b3c6b)
